packages/nexrender-api/src/job.js

```diff
@@ -48,6 +48,14 @@ module.exports = (fetch, polling) => ({
 
         return fetch(endpoint)
     },
+    loadJobDetails: async (id, data) => {
+        return fetch(`/jobs/${id}/details`, {
```

Comment:
Trying to understand that's happening here.

This looks like it's making a request against the automation server API, not the GraphQL API to get the cancelled status.

Reply:
Ah, related PR. I see now.

Reply:
nexrender worker talks only with nexrender server, while nexrender server talks with graphql api